### DISCUSSION TITLE
Aspect-ratio doesn't work while manually setting aspect width and height produces strange results

### DISCUSSION THREAD

**Discussion by pensivedog** [tailwindlabs/tailwindcss#11634] [Help] [answered]
`aspect-square` and `aspect-video` are not working for me at all--they simply don't have any effect.

Manually setting the aspect width and height does something, but not what expected. For instance, if I set `aspect-w-16` and `aspect-h-9` on an element, it does not produce something with a 16/9 aspect ratio--it is way, way off.

Can anyone please help explain what might be causing these issues?

**Comment by wongjn** [comment]
Please consider producing a [Tailwind Play](https://play.tailwindcss.com) that reproduces the unexpected behavior.

**Reply by pensivedog** [reply]
Thank you for the reply. Here is a Tailwind Play demonstrating my issue:

https://play.tailwindcss.com/3qGjj3LQ62

I'm simply trying to get an image to have a 16/9 aspect ratio, which is easy to do with normal CSS. What am I missing about how to do this with Tailwind CSS?

**Reply by wongjn** [reply, marked as answer]
Consider defining some sort of `width` on the image: https://play.tailwindcss.com/dnGzwZn8NK

**Reply by pensivedog** [reply]
That is very helpful. Thank you!
